### DISCUSSION TITLE
Possibility / ideas to lift a photo slide

### DISCUSSION THREAD

**Discussion by Stanislavmin** [igordanchenko/react-photo-album#140] [Q&A] [answered]
Hello, I got the photo album + YARL combination setup and rolling just great and what a great combination those two make, thank you!

There are certain type of photo slides I would love to lift and highlight more then others. Is there a good way to make the slide take more space / highlight it? Any ideas? Like taking up space of 4 objects rather then 1 slot and spanning over several columns / rows? Or any other good way anyone can think of? All tips appriciated. 

Kindly,
Stanislav

**Comment by igordanchenko** [comment, marked as answer]
Hi there!

I'm sorry to disappoint you, but unfortunately, this library isn't tooled to accommodate this use case. All supported layout algorithms are outlined in the library description, and there is no way to tell the library to span a particular photo across multiple columns or rows.

However, here are some ideas that you may find helpful:

1) Implement some kind of visual frame around the featured photo(s)
2) Add visual glow around the featured photo(s)
3) Manually render the first photo as a featured photo and then use two PhotoAlbum components to render photos to the right of the featured photo and below it. Here is a mock-up of this idea:

<img width="475" alt="Screenshot 2023-10-23 at 12 30 06 PM" src="https://github.com/igordanchenko/react-photo-album/assets/64441155/2ce9146c-c223-445d-83b7-b38fecdd23e4">

Here is a draft implementation illustrating this potential approach - https://codesandbox.io/p/sandbox/react-photo-album-140-z6k2l3?file=%2Fsrc%2FApp.tsx

I hope this helps!

**Reply by Stanislavmin** [reply]
Thank you, great sketch, that can absolutely be an option to solve the need!
